docs/template_plugin/tests/functional/op_reference/log_softmax.cpp

```diff
@@ -0,0 +1,253 @@
+using namespace InferenceEngine;
+
+namespace {
+struct LogSoftmaxParams {
+    template <class IT>
+    LogSoftmaxParams(const PartialShape& shape, const element::Type& iType, const std::vector<IT>& iValues, const std::vector<IT>& oValues,
+                const int64_t axis)
+        : axis(axis),
+          pshape(shape),
+          inType(iType),
+          outType(iType),
+          inputData(CreateBlob(iType, iValues)),
+          refData(CreateBlob(iType, oValues)) {}
+
+    int64_t axis = 0;
+
+    PartialShape pshape;
+    element::Type inType;
+    element::Type outType;
+    Blob::Ptr inputData;
+    Blob::Ptr refData;
+};
+
+class ReferenceLogSoftmaxLayerTest : public testing::TestWithParam<LogSoftmaxParams>, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = CreateFunction(params.pshape, params.inType, params.outType, params.axis);
+        inputData = {params.inputData};
+        refOutData = {params.refData};
+    }
+    static std::string getTestCaseName(const testing::TestParamInfo<LogSoftmaxParams>& obj) {
+        auto param = obj.param;
+        std::ostringstream result;
+        result << "shape=" << param.pshape << "_";
+        result << "iType=" << param.inType << "_";
+        result << "oType=" << param.outType << "_";
+        result << "axis=" << param.axis;
+        return result.str();
+    }
+
+private:
+    static std::shared_ptr<Function> CreateFunction(const PartialShape& input_shape, const element::Type& input_type,
+                                                    const element::Type& expected_output_type, const int64_t axis) {
+        const auto in = std::make_shared<op::Parameter>(input_type, input_shape);
+        const auto LogSoftmax = std::make_shared<op::v5::LogSoftmax>(in, axis);
+        return std::make_shared<Function>(NodeVector {LogSoftmax}, ParameterVector {in});
+    }
+};
+
+TEST_P(ReferenceLogSoftmaxLayerTest, CompareWithRefs) {
+    Exec();
+}
+
+template <element::Type_t IN_ET>
+std::vector<LogSoftmaxParams> generateLogSoftmaxFloatParams() {
+    using T = typename element_type_traits<IN_ET>::value_type;
+
+    std::vector<LogSoftmaxParams> logSoftmaxParams {
+        LogSoftmaxParams(ngraph::PartialShape {1},
+                    IN_ET,
+                    std::vector<T>{1},
+                    std::vector<T>{0},
+                    0),
+        LogSoftmaxParams(ngraph::PartialShape {2, 4},
+                    IN_ET,
+                    std::vector<T>{0, 1, 2, 3, 10000, 10001, 10002, 10003},
+                    std::vector<T>{-10000., -10000., -10000., -10000., 0., 0., 0., 0.},
+                    0),
+        LogSoftmaxParams(ngraph::PartialShape {2, 4},
+                    IN_ET,
+                    std::vector<T>{0, 1, 2, 3, 10000, 10001, 10002, 10003},
+                    std::vector<T>{-3.4401896, -2.4401896, -1.4401897, -0.4401897, -3.4401896, -2.4401896, -1.4401897, -0.4401897},
```

Comment:
@ilyachur @iefode one element(10003) passes in fp32, but fails in fp16. So I removed fp16 in test. Do I need to create JIRA ticket for fp16 accuracy issue so  template plugin owner make it fixed? I found that adding in skip_tests_config.cpp doesn't affect to op reference test cases.